src/referencePackages/src/system.data.sqlclient/4.8.5/ref/net6.0/System.Data.SqlClient.cs

```diff
@@ -4,354 +4,630 @@
-        public System.Threading.Tasks.Task WriteToServerAsync(System.Data.IDataReader reader) { throw null; }
-        public System.Threading.Tasks.Task WriteToServerAsync(System.Data.IDataReader reader, System.Threading.CancellationToken cancellationToken) { throw null; }
+
+        void IDisposable.Dispose() { }
+
+        public void WriteToServer(Common.DbDataReader reader) { }
+
+        public void WriteToServer(DataRow[] rows) { }
+
+        public void WriteToServer(DataTable table, DataRowState rowState) { }
+
+        public void WriteToServer(DataTable table) { }
+
+        public void WriteToServer(IDataReader reader) { }
+
+        public Threading.Tasks.Task WriteToServerAsync(Common.DbDataReader reader, Threading.CancellationToken cancellationToken) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(Common.DbDataReader reader) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(DataRow[] rows, Threading.CancellationToken cancellationToken) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(DataRow[] rows) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(DataTable table, DataRowState rowState, Threading.CancellationToken cancellationToken) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(DataTable table, DataRowState rowState) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(DataTable table, Threading.CancellationToken cancellationToken) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(DataTable table) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(IDataReader reader, Threading.CancellationToken cancellationToken) { throw null; }
+
+        public Threading.Tasks.Task WriteToServerAsync(IDataReader reader) { throw null; }
     }
+
     public sealed partial class SqlBulkCopyColumnMapping
     {
         public SqlBulkCopyColumnMapping() { }
+
         public SqlBulkCopyColumnMapping(int sourceColumnOrdinal, int destinationOrdinal) { }
+
         public SqlBulkCopyColumnMapping(int sourceColumnOrdinal, string destinationColumn) { }
+
         public SqlBulkCopyColumnMapping(string sourceColumn, int destinationOrdinal) { }
+
         public SqlBulkCopyColumnMapping(string sourceColumn, string destinationColumn) { }
+
         public string DestinationColumn { get { throw null; } set { } }
+
         public int DestinationOrdinal { get { throw null; } set { } }
+
         public string SourceColumn { get { throw null; } set { } }
+
         public int SourceOrdinal { get { throw null; } set { } }
     }
-    public sealed partial class SqlBulkCopyColumnMappingCollection : System.Collections.CollectionBase
+
+    public sealed partial class SqlBulkCopyColumnMappingCollection : Collections.CollectionBase
     {
-        internal SqlBulkCopyColumnMappingCollection() { }
```

Comment:
@andriipatsula it looks like GenAPI drops the internal ctor: https://github.com/dotnet/corefx/blob/b86ce69c316b4b088356b89bab80f7fa79ff799c/src/System.Data.SqlClient/ref/System.Data.SqlClient.cs#L313

Without the internal ctor, an object can be instantiated. Filed https://github.com/dotnet/sdk/issues/31776